nab/labeler.py

```diff
@@ -188,15 +192,14 @@ def write(self, destPath):
 
 
   def combine(self):
-    """Combine UserLabels."""
+    """Combine the raw user labels, and set the relaxed anomaly windows."""
     self.getUserLabels()
-    self.combineLabels()
-    self.combineWindows()
+    self.combineRawLabels()
     self.relaxWindows()
 
 
   def getUserLabels(self):
-    """Collect UserLabels."""
+    """Collect the raw user labels from the path 'nab/labels/raw/'."""
```

Comment:
It looks in the directory for "labels/raw". This is in scripts/combine_labels.py line 81. This is the organization for label files we want, no (issue#57)?